src/core_plugins/input_control_vis/public/components/vis/form_row.js

```diff
@@ -16,9 +17,24 @@ export function FormRow(props) {
     );
   }
 
+  let label = props.label;
```

Comment:
Might consider moving the conditional out into a helper function:

```js
function renderLabel({ label, control }) {
  if (!control.warning || control.warning.length === 0) {
    return label;
  }

  return (
    <Fragment>
      {label}
      <EuiIconTip
        content={control.warning}
        position="right"
        type="alert"
        aria-label="Warning"
      />
    </Fragment>
  );
}
```

So this would become:

```js
  return (
    <EuiFormRow
      label={renderLabel(props)}
      id={props.id}
      data-test-subj={'inputControl' + props.controlIndex}
    >
      {control}
    </EuiFormRow>
  );
```

Same goes for the `let control = props.children` section.